time time_stamp
string object_name
float64 distance
